docs/user_guide.rst

```diff
@@ -259,6 +259,47 @@ optimizations. Let us try it out on our decomposed and routed circuit above:
     gates for a particular device. In the example above we don't have them, however it is generally a good idea to run
     decomposition once again after the simplification.
 
+Clasically Controlled Gates
+----------------------------
+Some IQM quantum computers support classically controlled gates, that is, gates that are executed
```

Comment:
```suggestion
Classical control
-----------------
Some IQM quantum computers support classically-controlled gates, that is, gates that are executed
```

Just to follow the conventions of Cirq docs https://quantumai.google/cirq/build/classical_control